src/views/contact-us/contact-us.jsx

```diff
@@ -34,93 +35,62 @@ class ContactUs extends React.Component {
         } else if (query.indexOf('confirmation=') !== -1) {
             this.state.subject = 'Problem with email confirmation';
         }
+        // older iPads (iOS 10 or lower) cannot show help widget, show the form instead
+        if (bowser.osname === 'iOS' && parseInt(bowser.osversion, 10) < 11) {
```

Comment:
Nit: Since this won't change, you could make `showForm` a prop or even a constant to simplify rendering a bit.